server/graphql/v2/object/PaymentMethod.js

```diff
@@ -69,6 +70,25 @@ export const PaymentMethod = new GraphQLObjectType({
           return dataSubset;
         },
       },
+      limitedToHosts: {
+        type: new GraphQLList(Host),
+        async resolve(paymentMethod, args, req) {
+          let hosts;
+          if (paymentMethod.type === 'prepaid') {
+            const hostId = get(paymentMethod, 'data.HostCollectiveId', null);
+            if (!hostId) {
+              return;
+            }
+            const host = await req.loaders.Collective.byId.load(hostId);
+            hosts = [host];
+          } else if (paymentMethod.type === 'virtualcard' && paymentMethod.limitedToHostCollectiveIds) {
+            hosts = paymentMethod.limitedToHostCollectiveIds.map(id => {
+              return req.loaders.Collective.byId.load(id);
+            });
```

Comment:
Not a request change, but this can be simplified with the `loadMany` helper

```es6
hosts = req.loaders.Collective.byId.loadMany(paymentMethod.limitedToHostCollectiveIds);
```